src/Microsoft.DotNet.Tools.New/Template/NuGet.Config

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <!--To inherit the global NuGet package sources remove the <clear/> line below -->
+    <clear />
+    <add key="dotnet-core" value="https://www.myget.org/F/dotnet-core/api/v3/index.json" />
```

Comment:
Thanks @brthor it might be worth adding a tracking bug for fixing this before we ship.